x/exchange/params.go

```diff
@@ -0,0 +1,66 @@
+package exchange
+
+import (
+	"errors"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	// DefaultDefaultSplit is the default value used for the DefaultSplit parameter.
+	// TODO[1658]: Discuss what this should be with someone who would know.
+	DefaultDefaultSplit = uint32(500)
```

Comment:
Anyone have any thoughts on this value?

Reply:
After some offline discussion, we think this is a decent value to have until we know better. At which point, it'll probably already be in use and will be updated with a param change.